nautobot/docs/administration/nautobot-server.md

```diff
@@ -383,3 +383,13 @@ Listening on port http://localhost:9000. Stop with CONTROL-C.
 ```
 
 Please see the guide on [Troubleshooting Webhooks](../models/extras/webhook.md#troubleshooting) for more information.
+
+### `startplugin`
```

Comment:
Looks like this file lists commands alphabetically, so please move this section up as appropriate.